test/tests/utilities_internalTest.js

```diff
@@ -569,6 +569,45 @@ describe("Zotero.Utilities.Internal", function () {
 			assert.equal(out, '"" has a length of 0 and "hello" has a length of 5');
 		});
 
+		it("should support functions in comparison statements", function () {
+			const vars = {
+				sum: ({ a, b }) => parseInt(a) + parseInt(b),
+				fahrenheitToCelsius: ({ temp }) => Math.round((temp - 32) * 5 / 9),
+				false: 'false',
+				twoWords: 'two words',
+				onlyOne: 'actually == 1'
+			};
+			const template = `{{if sum a=1 b=2 == 3}}1 + 2 = {{sum a=1 b=2}}{{else}}no speak math{{endif}}`;
```

Comment:
This is kind of awkward and hard to read. Do we want to require some kind of delimiter when using parameters within conditions? An extra set of braces? Parens?

`{{if {{sum a=1 b=2}} == 3}}` (more consistent)

`{{if (sum a=1 b=2) == 3}}` (more concise, but introducing additional syntax)

Do we even need to support parameters in conditionals?

Reply:
> Do we even need to support parameters in conditionals?

We don't strictly need it, for the `itemType` case for this PR we just need to be able to use a function inside logical statement. However, outside of this very specific case, functions without parameters offer very limited functionality. I'd rather go with the nested `{{ }}` but I can remove support for parameters if you think we shouldn't support that?

Reply:
I guess we can go with nested `{{ }}` in case we need it later and just not document it now (since there are no functions with parameters to document).